app/components/gh-members-lab-setting.js

```diff
@@ -26,9 +26,14 @@ export default Component.extend({
     config: service(),
     mediaQueries: service(),
 
-    currencies: currencies,
+    currencies: null,
```

Comment:
Even though this is a constant its best practice to never use arrays/objects as default properties in an EmberObject because the values can end up being shared across instances and leaking state.

We have a lint rule to protect against it, not sure why it didn't pick up this case 🤔 https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/avoid-leaking-state-in-ember-objects.md